packages/flutter/lib/src/widgets/form.dart

```diff
@@ -530,7 +530,7 @@ class FormField<T> extends StatefulWidget {
   ///
   /// Defaults to [AutovalidateMode.disabled].
   /// {@endtemplate}
-  final AutovalidateMode autovalidateMode;
+  final AutovalidateMode autoValidateMode;
```

Comment:
Technically, we could further mitigate breakages with something like

```dart
  @Deprecated(...)
  AutovalidateMode get autovalidateMode => autoValidateMode;

  final AutovalidateMode autoValidateMode;
```

but if checks are passing as-is, then it's not necessary.